A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Library.xml

```diff
@@ -1,1275 +1,1360 @@
+         <Instrument Id="ENGINE_Lever_Throttle" />
+      </Checkpoint>
+      <Checkpoint Id="A320_LANDING_AT_TOUCHDOWN_SPACER">
+         <CheckpointDesc SubjectTT="TT:AT TOUCHDOWN" ExpectationTT="TT:" />
+      </Checkpoint>
+      <Checkpoint Id="A320_LANDING_ENGINE_REVERSE">
+         <CheckpointDesc SubjectTT="TT:Engine Reversers" ExpectationTT="TT:MAX" />
+      </Checkpoint>
+      <Checkpoint Id="A320_LANDING_GRND_SPOILERS">
+         <CheckpointDesc SubjectTT="TT:Ground Spoilers" ExpectationTT="TT:CHECK" />
+         <Instrument Id="HANDLING_Lever_Spoilers" />
+      </Checkpoint>
+      <Checkpoint Id="A320_LANDING_DIRECTIONAL_CONTROL">
+         <CheckpointDesc SubjectTT="TT:Directional Control" ExpectationTT="TT:CHECK" />
+      </Checkpoint>
+      <Checkpoint Id="A320_LANDING_BRAKES">
+         <CheckpointDesc SubjectTT="TT:Brakes" ExpectationTT="TT:AS REQUIRED" />
+      </Checkpoint>
+      <Checkpoint Id="A320_LANDING_AT_70KTS_SPACER">
+         <CheckpointDesc SubjectTT="TT:AT 70KTS" ExpectationTT="TT:" />
+      </Checkpoint>
+      <Checkpoint Id="A320_LANDING_THRUST_LEVERS_REVERSE_IDLE">
+         <CheckpointDesc SubjectTT="TT:Thrust Levers" ExpectationTT="TT:REVERSE IDLE" />
+         <Instrument Id="ENGINE_Lever_Throttle" />
+      </Checkpoint>
+      <Checkpoint Id="A320_LANDING_THRUST_LEVERS_IDLE">
+         <CheckpointDesc SubjectTT="TT:Thrust Levers" ExpectationTT="TT:IDLE (AT 25KTS)" />
+         <Instrument Id="ENGINE_Lever_Throttle" />
+      </Checkpoint>
+      <Checkpoint Id="A320_LANDING_AUTOBRAKES">
+         <CheckpointDesc SubjectTT="TT:Autobrakes" ExpectationTT="TT:DISENGAGE BELOW 20KTS" />
+      </Checkpoint>
+      <!-- AFTER LANDING -->
+      <Checkpoint Id="A320_AFTERLANDING_LANDING_LIGHTS">
+         <CheckpointDesc SubjectTT="TT:Landing Lights" ExpectationTT="TT:OFF" />
+      </Checkpoint>
+      <Checkpoint Id="A320_AFTERLANDING_GRND_SPOILERS">
+         <CheckpointDesc SubjectTT="TT:Ground Spoilers" ExpectationTT="TT:DISARM" />
+         <Instrument Id="HANDLING_Lever_Spoilers" />
+      </Checkpoint>
+      <Checkpoint Id="A320_AFTERLANDING_FLAPS">
+         <CheckpointDesc SubjectTT="TT:Flaps" ExpectationTT="TT:RETRACTED" />
+         <Instrument Id="HANDLING_Lever_Flaps" />
+      </Checkpoint>
+      <Checkpoint Id="A320_AFTERLANDING_ENG_MODE_SEL">
+         <CheckpointDesc SubjectTT="TT:Engine Mode Selector" ExpectationTT="TT:NORM" />
+         <Instrument Id="ENGINE_Switch_Engine_Mode" />
+      </Checkpoint>
+      <Checkpoint Id="A320_AFTERLANDING_ATC">
+         <CheckpointDesc SubjectTT="TT:ATC" ExpectationTT="TT:STBY/OFF" />
+      </Checkpoint>
+      <Checkpoint Id="A320_AFTERLANDING_TCAM_MODE_SEL">
+         <CheckpointDesc SubjectTT="TT:TCAM Mode Selector (INOP)" ExpectationTT="TT:STBY" />
+      </Checkpoint>
+      <Checkpoint Id="A320_AFTERLANDING_RADAR">
+         <CheckpointDesc SubjectTT="TT:Radar" ExpectationTT="TT:OFF/STBY" />

```

Comment:
I changed this to Weather Radar with expectation OFF (there is no STBY?)

Reply:
SYS on with Mode Off = STBY perhaps?

Reply:
The OFF mode is INOP sadly, but that could be the case indeed. Just above is STDBY/OFF so it feels a bit inconsistent, and the Radar > Weather Radar change is nice to do everywhere consistently I think though :)